src/Uno.UI/UI/Xaml/Media/LinearGradiientBrush.faux.cs

```diff
@@ -0,0 +1,120 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Uno.UI.Xaml.Media;
+using Windows.Foundation.Collections;
+using Windows.UI;
+
+namespace Microsoft.UI.Xaml.Media;
+
+public partial class LinearGradientBrush
+{
+	private Color? _majorStopColor;
+	private SolidColorBrush? _fauxOverlayBrush;
+
+	/// <summary>
+	/// Gets or sets a flag indicating whether this brush should use
+	/// major gradient solid brush color in case the target platform
+	/// cannot apply the linear gradient to border correctly.
+	/// </summary>
+	public bool SupportsFauxBorder
+	{
+		get => (bool)GetValue(SupportsFauxBorderProperty);
+		set => SetValue(SupportsFauxBorderProperty, value);
+	}
+
+	/// <summary>
+	/// Identifies the SupportsFauxBorder dependency property.
+	/// </summary>
+	public static DependencyProperty SupportsFauxBorderProperty { get; } =
+		DependencyProperty.Register(
+			nameof(SupportsFauxBorder),
+			typeof(bool),
+			typeof(LinearGradientBrush),
+			new FrameworkPropertyMetadata(false));
+
+	internal Color? MajorStopColorWithOpacity => _majorStopColor ??= GetMajorStop()?.Color.WithOpacity(Opacity);
+
+	internal SolidColorBrush? FauxOverlayBrush => _fauxOverlayBrush ??= CreateFauxOverlayBrush();
+
+	internal override bool CanApplyToBorder(CornerRadius cornerRadius)
```

Comment:
I wonder if this method should be under `#if __WASM__ || __IOS__ || __MACOS__`

Reply:
Yes, the whole file is now `#ifd`